generator/src/lib.rs

```diff
@@ -1264,13 +1264,18 @@ pub struct ExtensionCommands<'a> {
 }
 
 pub fn generate_extension_commands<'a>(
-    full_extension_name: &'a str,
-    items: &'a [vk_parse::ExtensionChild],
+    extension: &'a vk_parse::Extension,
     cmd_map: &CommandMap<'a>,
     cmd_aliases: &HashMap<&'a str, &'a str>,
     fn_cache: &mut HashSet<&'a str>,
     has_lifetimes: &HashSet<Ident>,
 ) -> ExtensionCommands<'a> {
+    let vk_parse::Extension {
+        name: full_extension_name,
+        children: items,
+        promotedto,
```

Comment:
`promoted_to` ?

Reply:
This is defined in vk_parse. We can probably rename it to promoted_to in our code if that makes you happy